src/projects/project-pull-requests.html

```diff
@@ -125,16 +125,30 @@
         hideProjectButton: {
           type: String,
           value: ''
+        },
+        getFields: {
+          type: Function,
+          value: function() {
+            return function(e) {
```

Comment:
I think the argument name could be made more explicit to reflect that it is a pull request object.